logical/plugin/backend_client.go

```diff
@@ -117,10 +117,9 @@ func (b *backendPluginClient) HandleRequest(req *logical.Request) (*logical.Resp
 		if reply.Error.Error() == logical.ErrUnsupportedOperation.Error() {
 			return nil, logical.ErrUnsupportedOperation
 		}
-		return nil, reply.Error
```

Comment:
I think you might need to do `return reply.Response, reply.Error` here, and keep the other line to explicitly return nil